Formula/b/basti.rb

```diff
@@ -0,0 +1,21 @@
+require "language/node"
+
+class Basti < Formula
+  desc "Cloud credentials manager cli"
+  homepage "https://github.com/basti-app/basti"
+  url "https://registry.npmjs.org/basti/-/basti-1.6.2.tgz"
+  sha256 "8d813c1f4e3b8195655d40e670aa8a2eb7dce2cd21d996564c56a3296163f1d7"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_match "#compdef basti",
+      shell_output("#{bin}/basti completion").strip
```

Comment:
I don't know if this will help us detect if the software install is broken. Is this a command that users will often run?

Reply:
completion should be actually part of install steps.

Reply:
> Would you have a suggestion on what can be run? The other commands seem to require AWS access or throw an error.

Checking for the error seems like a good option

Reply:
I can add the completion as part of the install.

The issue with checking for an error is that if an AWS_PROFILE or keys are set, it's possible it runs aws api calls when the test runs (upon installaton?) which could error out as the command is interactive or if the interactivity is solved, it could also create/destroy resources.

One way we could avoid this is by setting a randomly generated string for the aws profile and then the command would simply fail on the missing aws profile. Would that be better?

Reply:
> One way we could avoid this is by setting a randomly generated string for the aws profile and then the command would simply fail on the missing aws profile. Would that be better?

that works, I think you can find some examples in the repo :)

Reply:
> The issue with checking for an error is that if an AWS_PROFILE or keys are set, it's possible it runs aws api calls

Brew tests run in a very cleaned environment 

> when the test runs (upon installaton?) 

When a user runs `brew test <formula>`

> One way we could avoid this is by setting a randomly generated string for the aws profile and then the command would simply fail on the missing aws profile. Would that be better?

Sure, you can do that